core/client/package.json

```diff
@@ -34,6 +34,7 @@
     "ember-cli-simple-auth-oauth2": "0.8.0-beta.2",
     "ember-cli-uglify": "1.0.1",
     "ember-data": "1.0.0-beta.18",
+    "ember-disable-proxy-controllers": "^0.7.0",
```

Comment:
I think you will need to remove this unless you have removed all usage of `itemController` from the app.  Otherwise you will get super cryptic "Expected controller:array to be an ArrayController" error message.